custom_components/e3dc_rscp/select.py

```diff
@@ -0,0 +1,84 @@
+from dataclasses import dataclass
+from typing import Final, Awaitable, Callable
+
+from homeassistant.components.select import (
+    SelectEntity,
+    SelectEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from custom_components.e3dc_rscp import const, E3DCCoordinator
+
+from .const import DOMAIN
+
+OPTIONS: Final[dict[str, dict[str, int]]] = {
```

Comment:
Can we add string to the powermodes / modes in some way, so that this also gets localizable? I haven'd used selects yet, but I'd assume that the options have translation support. 
It would be fantastic if you could give the docs a quick spin to check if we could add that to make the UX better.

Reply:
the select is removed. It was also some testing stuff. But I expect that the state in string.json can also be used to translate them